test/functional/symbols.json.example

```diff
@@ -1,6 +1,7 @@
 {
     "bigip_mgmt_ip": "10.190.7.183",
     "bigip_mgmt_ip_public": "10.190.7.183",
+    "bigip_ssh_username": "root",
```

Comment:
@richbrowne if you don't want the default symbols file to be updated to reflect this new required input, then you'll have to change the default behavior of the bigip-tracker feature (to make it "opt in", so it's not "on" by default).

With the current setup, if the bigip-tracker feature is enabled, then the bigip_ssh_username symbol is required.